### ISSUE TITLE
Deployment error when running "crmint cloud setup"

### ISSUE THREAD

**Issue by mariamunze** [google/crmint#132]
(venv) mariamunze@cloudshell:~$ crmint cloud setup
>>>> Setup
---> Get current project identifier  
     Project ID found:
Traceback (most recent call last):
  File "/home/mariamunze/crmint/venv/bin/crmint", line 11, in <module>
    load_entry_point('crmint', 'console_scripts', 'crmint')()
  File "/home/mariamunze/crmint/cli/appcli.py", line 108, in entry_point
    cli()
  File "/home/mariamunze/crmint/venv/local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/mariamunze/crmint/venv/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/mariamunze/crmint/venv/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/mariamunze/crmint/venv/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/mariamunze/crmint/venv/local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/mariamunze/crmint/venv/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/mariamunze/crmint/cli/commands/cloud.py", line 507, in setup
    stage_name, stage = fetch_stage_or_default(stage_name, debug=debug)
  File "/home/mariamunze/crmint/cli/commands/cloud.py", line 35, in fetch_stage_or_default
    stage = shared.get_stage_object(stage_name)
  File "/home/mariamunze/crmint/cli/utils/shared.py", line 83, in get_stage_object
    return getattr(__import__("stages.%s" % stage_name), stage_name)
AttributeError: 'module' object has no attribute ''

**Comment by Slony** [comment]
@mariamunze looks like you've had no Cloud Project selected.

Please, run the following command to select a Cloud Project before setting up the cloud environment required for CRMint:

```bash
gcloud config set project your-project-id
```

Don't forge to replace `your-project-id` with the real project ID.

Then consider running these commands to deploy CRMint:

```bash
# Change to CRMint's directory.
cd ~/crmint

# Activate Python's virtual environment.
. venv/bin/activate

# Create a stage definition for your environment.
crmint stages create

# Re-run the setup in case new elements are needed.
crmint cloud setup

# Deploy the updated App Engine services.
crmint cloud deploy
```

Please, let me know if all of the above works for you.

Cheers!

**Comment by mariamunze** [comment]
@Slony  thank you very much :+1: 
I managed to deploy the app with docker as well as with venv in google cloud but in both cases I get an error in the main crmint page, saying:  **There is error after loading data.** 
I can not see the Dashboard tab, only the Pipeline and Settings tabs. Also I can not see the "Service Account Email" in the settings tab, although I have copied the service account key in the **./backends/data/service-account.json**

Locally it works fine. I copied the service accout key into the **./backends/data/service-account.json** folder. and I am able to create pipelines.
Am I missing something?

Thank you very much.
